src/kinetics-observer.cpp

```diff
@@ -130,10 +131,13 @@ KineticsObserver::KineticsObserver(unsigned maxContacts, unsigned maxNumberOfIMU
   ekf_.setFunctor(this);
   ekf_.setStateArithmetics(this);
 
-  stateVector_.setZero();
-  oldStateVector_ = stateVector_;
+  /*  The initialization of the observer is now called by MCKineticsObserver
+  worldCentroidStateVector_.setZero();
+  oldWorldCentroidStateVector_ = worldCentroidStateVector_;
 
-  ekf_.setState(stateVector_, k_est_);
+  ekf_.setState(worldCentroidStateVector_, k_est_);
+  updateKine_();
+  */
```

Comment:
This is a little unclear, does it mean that after creating this observer object we need to initialize it in a specific way? If so this needs to be better documented

Reply:
This comment was actually unnecessary, I discarded this change to use the original initialization. I added a comment to say that we can initialize the state vector after the call of this function (this is what we do in MCKineticsObserver).